guidelines/index.html

```diff
@@ -13,7 +13,7 @@
     </head>
     <body>
         <section id="abstract">
-            <p>Web Content Accessibility Guidelines (WCAG) 2.2 covers a wide range of recommendations for making Web content more accessible. Following these guidelines will make content more accessible to a wider range of people with disabilities, including accommodations for blindness and low vision, deafness and hearing loss, limited movement, speech disabilities, photosensitivity, and combinations of these, and some accommodation for learning disabilities and cognitive limitations; but will not address every user need for people with these disabilities. These guidelines address accessibility of web content on desktops, laptops, tablets, and mobile devices. Following these guidelines will also often make Web content more usable to users in general.</p>
+            <p>Web Content Accessibility Guidelines (WCAG) 2.2 covers a wide range of recommendations for making Web content more accessible. Following these guidelines will make content more accessible to a wider range of people with disabilities, including accommodations for blindness and low vision, deafness and hearing loss, limited movement, speech disabilities, photosensitivity, and combinations of these, and some accommodation for learning disabilities and cognitive limitations; but will not address every user need for people with these disabilities. These guidelines address accessibility of web content regardless of device (such as desktops, laptops, and mobile devices). Following these guidelines will also often make Web content more usable to users in general.</p>
```

Comment:
"on any" seems simpler than "regardless of". And there are a huge range of devices people use in reality that present web content, includng ATMs, information kiosks, watches, those spyng machines in your house that you talk to, TVs, etc. In addition, these things keep getting blended into each other.  **IMHO** the less we try to list devices, the better...

```suggestion
            <p>Web Content Accessibility Guidelines (WCAG) 2.2 covers a wide range of recommendations for making Web content more accessible. Following these guidelines will make content more accessible to a wide range of people with disabilities, including accommodations for blindness and low vision, deafness and hearing loss, limited movement, speech disabilities, photosensitivity, and combinations of these, and some accommodation for learning disabilities and cognitive limitations; but will not address every user need for people with these disabilities. These guidelines address accessibility of web content on any kind of device (including desktop snd laptop computers, mobile devices, and kiosk devices). Following these guidelines will also often make Web content more usable to users in general.</p>
```

Reply:
I fixed a typo in your suggestion. I wonder if it might work better as just "...(including desktops, laptops, kiosks, and mobile devices)."

That seems to cover very broad categories of systems, with few words changed.

Reply:
```suggestion
            <p>Web Content Accessibility Guidelines (WCAG) 2.2 covers a wide range of recommendations for making Web content more accessible. Following these guidelines will make content more accessible to a wider range of people with disabilities, including accommodations for blindness and low vision, deafness and hearing loss, limited movement, speech disabilities, photosensitivity, and combinations of these, and some accommodation for learning disabilities and cognitive limitations; but will not address every user need for people with these disabilities. These guidelines address accessibility of web content on any kind of device (including desktops, laptops, kiosks, and mobile devices). Following these guidelines will also often make Web content more usable to users in general.</p>
```